index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <nav class="toolbar">
+            <a class="logo">
+                My Logo
+            </a>
```

Comment:
- [ ] Please I highly recommend adding an `href` to the `a` tag to make `My Logo` have a link to a specific section and implement the convention, HTML semantics and better user experience. This is a provided link for more [information](https://developer.mozilla.org/en-US/docs/Glossary/Semantics)